### ISSUE TITLE
Add option to remove unneccesary icons from the world map

### ISSUE THREAD

**Issue by ShaunDreclin** [runelite/runelite#5386]
I think the main ones worth having are dungeons/dungeon links, altars, transportation, agility shortcuts, banks, and teleport destinations.

**Comment by sirlaige** [comment]
Yes, I hate the clutter.  I'm not sure why were need a voting station icon for example when almost all banks have a voting stations.

**Comment by raiyni** [comment]
Because it's a jagex icon.

**Comment by sirlaige** [comment]
Jagex icons can be remove easy enough.

**Comment by raiyni** [comment]
okay submit the pr then

**Comment by raiyni** [comment]
code it and find out
